static/js/redux/ui/comment_forum.jsx

```diff
@@ -14,19 +14,100 @@ GNU General Public License for more details.
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import AdvisorMenu from './advisor_menu';
 import CommentInputContainer from './containers/comment_input_container';
+import {getTranscriptCommentsBySemester} from '../constants/endpoints';
+import Cookie from 'js-cookie';
 
+let semester_name;
+let semester_year;
 
 class CommentForum extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        semester_name: '',
+        semester_year: '',
+        transcript: null,
+        comments: null,
+        addedAdvisors: [
+          'yamir',
+          'lmoulton'
+        ],
+        //TODO: Set this to list of student's advisors from SIS
```

Comment:
Create an issue for this so we can track it. This would involve a fetch() to get the data from the backend (a list of the student's advisors)